plugins/toolkit/jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/rds/auth/IamAuth.kt

```diff
@@ -49,6 +58,17 @@ class IamAuth : DatabaseAuthProviderCompatabilityAdapter {
 
     override fun createWidget(project: Project?, credentials: DatabaseCredentials, dataSource: LocalDataSource): AuthWidget? = IamAuthWidget()
 
+    inner class SsoNoTokenFix(val project: Project, val connection: ProtoConnection) : ErrorInfo.Fix {
```

Comment:
does this really need to be an inner class

Reply:
we are using `getAuthInformation` which is not accessible without using inner class.